code/style.css

```diff
@@ -1,4 +1,279 @@
+/* Mobile first*/
+
 html {
-  background: #0025ff;
-  color: #fff;
+  background: #e4eef3;
+  color: rgb(56, 54, 54);
+  font-family: 'Urbanist', sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  padding: 20px;
+}
+
+.navigation {
+  color:black;
+  font-size: 15px;
+  margin-right: 5px;
+}
+
+a {
+text-decoration: none;
+margin-right: 20px;
+color: black;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+.logo {
+  width: 100px;
+  border-radius: 50%;
+}
+
+.site-name {
+  display: none;
+}
+
+h1 {
+  color:black;
+  font-size: 50px;
+}
+
+h1:hover {
+  color: rgb(112, 60, 233);
+}
+
+/* big news style */
+
+.big-news {
+  display: flex;
+  flex-direction: column;
+  border: 2px solid rgb(214, 209, 209);
+  background-color:rgb(243, 239, 239);
+}
+
+.big-news-image {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 100%;
+}
+
+.big-news-text {
+  padding: 10px 5px;
+}
+
+/* other news style */
+
+.other-news {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.card {
+  width: 100%;
+  background-color: white;
+  padding: 5px;
+  margin: 20px auto;
+  box-shadow: 4px 8px 8px 0 rgba(0, 0, 0, 1), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  text-align:inherit;
+} 
+
+.card-pic {
+  width: 100%;
+  background-color: white;
+  padding: 0 0 5 0px;
+  margin: 20px auto;
+  box-shadow: 4px 8px 8px 0 rgba(0, 0, 0, 1), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  text-align: center;
+}
+
+.container-text {
+  padding: 10px;
+}
+
+button {
+  background-color: rgb(233, 184, 121);
+  color: white;
+  border: 2px solid gray;
+  border-radius: 16px;
+  padding: 10px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  transition-duration: 0.6s;
+  cursor: pointer;
+}
+
+button:hover {
+  background-color: rgb(134, 197, 134);
+  font-size: 20px;
+  text-decoration: none;
+}
+
+.navigation {
+  display: none;
+}
+
+img {
+  max-width:100%;
+}
+
+footer {
+  border-top: 2px solid gray;
+  padding: 20px;
+  background-color: #e4eef3;
+  text-align: right;
+}
+
+/* Tablet style */
+
+@media (min-width:768px) {
```

Comment:
Smart to style only min-width so that you don't have to repeat the same for desktop later on. Nice!!